lms/static/scripts/frontend_apps/components/LMSPagePicker.tsx

```diff
@@ -0,0 +1,300 @@
+import {
+  Button,
+  ModalDialog,
+  SpinnerOverlay,
+} from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import { useCallback, useEffect, useState } from 'preact/hooks';
+
+import type { Page } from '../api-types';
+import type { APICallInfo } from '../config';
+import { isAuthorizationError } from '../errors';
+import { apiCall } from '../utils/api';
+import AuthButton from './AuthButton';
+import ErrorDisplay from './ErrorDisplay';
+import ResourceList from './ResourceList';
+
+/**
+ * Renders a helpful message with a link to documentation when there are no
+ * pages.
+ *
+ * @todo set useful link
+ */
+function NoPagesMessage() {
+  return (
+    <div>
+      There are no pages in this instance.{' '}
+      <a href={'TODO'} target="_blank" rel="noreferrer">
```

Comment:
We need to add a proper link here like we do with files.

Reply:
I'll create a board for this soon and I'll a draft about this.

Reply:
Created here: https://github.com/orgs/hypothesis/projects/126?pane=issue&itemId=40446950